ksql-streams/src/main/java/io/confluent/ksql/execution/streams/IRoutingFilter.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.streams;
+
+import org.apache.kafka.streams.state.HostInfo;
+
+public interface IRoutingFilter {
```

Comment:
Add a comment explaining how this is used.

Also, I don't see any interfaces prefixed with "I" in the codebase, so maybe just "RoutingFilter"?

Reply:
Done